src/deno_dir.rs

```diff
@@ -346,6 +348,24 @@ impl DenoDir {
   }
 }
 
+impl SourceMapGetter for DenoDir {
+  fn get_source_map(&self, script_name: &str) -> Option<String> {
+    match self.code_fetch(script_name, ".") {
```

Comment:
I think this could lead to people getting very confused if the cache somehow gets updated after starting the script, but before getting the SourceMap.